README.md

```diff
@@ -1,128 +1,12 @@
-
-3. Build the default Fluo docker image using the command below.
-
-        cd /path/to/fluo-docker
-        docker build -t fluo .
-
-   Or build the Fluo docker image with specific versions of Hadoop, Zookeeper, etc using the command below:
-
-        docker build --build-arg ZOOKEEPER_VERSION=3.4.8 --build-arg ACCUMULO_VERSION=1.7.3 --build-arg HADOOP_VERSION=2.7.0 -t fluo .
-
-## Image basics
-
-The entrypoint for the Fluo docker image is the `fluo` script. While the primary use
-case for this image is to start an oracle or worker, you can run other commands in the
-`fluo` script to test out the image:
-
-```bash
-# No arguments prints Fluo command usage
-docker run mikewalch/fluo
-# Print Fluo version
-docker run mikewalch/fluo version
-# Print Fluo classpath
-docker run mikewalch/fluo classpath
-```
-
-# Run Fluo Applications using Docker
-
-Before starting a Fluo oracle and worker using the Fluo Docker image, [initialize your Fluo application][application]. 
-Next, choose a method below to run the oracle and worker(s) of your Fluo application. In the examples below, the Fluo
-application is named `myapp` and was initialized using a Zookeeper node on `zkhost`.
-
-## Docker engine
-
-Use the `docker` command to start local docker containers.
-
-1. Start a Fluo oracle
-
-        docker run -d --network="host" mikewalch/fluo oracle -a myapp -o fluo.connection.zookeepers=zkhost/fluo
-
-2. Start Fluo worker(s). Execute this command multiple times to start multiple workers.
-
-        docker run -d --network="host" mikewalch/fluo worker -a myapp -o fluo.connection.zookeepers=zkhost/fluo
-
-## Marathon
-
-Using the [Marathon] UI, you can create applications using JSON configuration.
-
-The JSON below can be used to start a Fluo oracle.
-
-```json
-{
-  "id": "myapp-fluo-oracle",
-  "cmd": "fluo oracle -a myapp -o fluo.connection.zookeepers=zkhost/fluo",
-  "cpus": 1,
-  "mem": 256,
-  "disk": 0,
-  "instances": 1,
-  "container": {
-    "docker": {
-      "image": "mikewalch/fluo",
-      "network": "HOST"
-    },
-    "type": "DOCKER"
-  }
-}
-```
-
-The JSON below can be used to start Fluo worker(s). Modify instances to start multiple workers.
-
-```json
-{
-  "id": "myapp-fluo-worker",
-  "cmd": "fluo worker -a myapp -o fluo.connection.zookeepers=zkhost/fluo",
-  "cpus": 1,
-  "mem": 512,
-  "disk": 0,
-  "instances": 1,
-  "container": {
-    "docker": {
-      "image": "mikewalch/fluo",
-      "network": "HOST"
-    },
-    "type": "DOCKER"
-  }
-}
-```
+* [Documentation] for running a Fluo application in Docker
 
 [Fluo]: https://fluo.apache.org/
-[Accumulo]: https://accumulo.apache.org/
-[Hadoop]: https://hadoop.apache.org/
-[Zookeeper]: https://zookeeper.apache.org/
-[application]: https://github.com/apache/fluo/blob/master/docs/applications.md
-[Marathon]: https://mesosphere.github.io/marathon/
+[Documentation]: https://fluo.apache.org/docs/fluo/1.2/administration/run-fluo-in-docker/
```

Comment:
Is it possible to have  a link w/o a version?

Reply:
It is but we would probably need to update that link with each release.